seqr/utils/logging_utils.py

```diff
@@ -22,10 +24,10 @@ def formatMessage(self, record):
         if hasattr(record, 'db_update'):
             log_json['dbUpdate'] = record.db_update
 
-        if hasattr(record, 'traceback'):
+        if getattr(record, 'traceback', None):
             log_json['traceback'] = record.traceback
 
-        if record.levelname == 'ERROR':
+        if record.levelname == 'ERROR' and DEPLOYMENT_TYPE != 'dev':
```

Comment:
Does `DEPLOYMENT_TYPE != 'dev'` mean `DEPLOYMENT_TYPE == 'prod'?  Is there any other valid type which logging on google?

Reply:
right now theres only dev and prod, but I thought it was better to alert on log errors by default and explicitly exclude the cases we don't want to be alerted (i.e for dev), rather than explicitly include what we do want. This means that if in the future we change the name of the deployment there is no risk of losing our error reporting - if we make a breaking change it would cause us to get more logs than we want rather than less, which is a safer problem to have